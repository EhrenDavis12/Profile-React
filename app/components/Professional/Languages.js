import React from 'react';
import { SectionBreak, Ul } from 'components/SharedStyles/styles';
import Languages from './Lists/Languages';

function TechnicalSkills() {
  const languages = Languages.map(skill => <li key={skill}>{skill}</li>);
  return (
    <div>
      <SectionBreak>
        <h4>Languages:</h4>
      </SectionBreak>
      <SectionBreak>
        <Ul>{languages}</Ul>
      </SectionBreak>
    </div>
  );
}

export default TechnicalSkills;
