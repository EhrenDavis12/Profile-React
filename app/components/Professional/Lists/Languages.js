export default [
  'React',
  'React-Native',
  'Redux',
  'Redux-Saga',
  '4D: 6 years',
  'T-SQL',
  'Regex',
  'Bash',
  'JavaScript',
  'RESTful APIs',
  'SOAP APIs',
  'VIM',
  'CronJobs',
  'XML',
  'JSON',
  'JQuery',
  'AJAX',
  'Jest',
  'Bootstrap',
  'HTML5',
  'Sequelize',
  'VBA',
  'Python',
  '.NET',
  'C#',
  'CSS',
  'C++',
];
