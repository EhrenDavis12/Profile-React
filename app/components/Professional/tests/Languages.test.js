import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Languages from '../Languages';

describe('<Languages />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<Languages />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
