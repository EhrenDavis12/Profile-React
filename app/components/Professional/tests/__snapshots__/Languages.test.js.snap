// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Languages /> should render the page 1`] = `
<div>
  <styles__SectionBreak>
    <h4>
      Languages:
    </h4>
  </styles__SectionBreak>
  <styles__SectionBreak>
    <styles__Ul>
      <li
        key="React"
      >
        React
      </li>
      <li
        key="React-Native"
      >
        React-Native
      </li>
      <li
        key="Redux"
      >
        Redux
      </li>
      <li
        key="Redux-Saga"
      >
        Redux-Saga
      </li>
      <li
        key="4D"
      >
        4D
      </li>
      <li
        key="T-SQL"
      >
        T-SQL
      </li>
      <li
        key="Regex"
      >
        Regex
      </li>
      <li
        key="Bash"
      >
        Bash
      </li>
      <li
        key="JavaScript"
      >
        JavaScript
      </li>
      <li
        key="RESTful APIs"
      >
        RESTful APIs
      </li>
      <li
        key="SOAP APIs"
      >
        SOAP APIs
      </li>
      <li
        key="VIM"
      >
        VIM
      </li>
      <li
        key="CronJobs"
      >
        CronJobs
      </li>
      <li
        key="XML"
      >
        XML
      </li>
      <li
        key="JSON"
      >
        JSON
      </li>
      <li
        key="JQuery"
      >
        JQuery
      </li>
      <li
        key="AJAX"
      >
        AJAX
      </li>
      <li
        key="Jest"
      >
        Jest
      </li>
      <li
        key="Bootstrap"
      >
        Bootstrap
      </li>
      <li
        key="HTML5"
      >
        HTML5
      </li>
      <li
        key="Sequelize"
      >
        Sequelize
      </li>
      <li
        key="VBA"
      >
        VBA
      </li>
      <li
        key="Python"
      >
        Python
      </li>
      <li
        key=".NET"
      >
        .NET
      </li>
      <li
        key="C#"
      >
        C#
      </li>
      <li
        key="CSS"
      >
        CSS
      </li>
      <li
        key="C++"
      >
        C++
      </li>
    </styles__Ul>
  </styles__SectionBreak>
</div>
`;
